scripts/docker/astrobee_base.Dockerfile

```diff
@@ -46,7 +44,7 @@ RUN apt-get update \
 COPY ./scripts/setup/debians /setup/astrobee/debians
 
 RUN apt-get update \
-  && /setup/astrobee/debians/build_install_debians.sh \
+  && /bin/bash /setup/astrobee/debians/build_install_debians.sh \
```

Comment:
The way you're switching to `bash` here may be the best, most conservative choice in this case, but I wanted to point out that Docker also allows you to switch from `sh` to `bash` for all `RUN` commands in the Dockerfile using this command:

    SHELL ["/bin/bash", "-c"]